addons/web/static/src/module_loader.js

```diff
@@ -170,7 +170,46 @@
             }
 
             domReady(() => {
-                // Empty body
+                    const customJSUnloaded = unloaded.some((module) =>
+                    module.includes("user_custom_javascript")
+                );
+
+                if (customJSUnloaded && (document.querySelector(".o_frontend_to_backend_edit_btn") || window.location.href.includes("/web/login"))) {
+                    if (document.querySelector(".custom-js-unloaded-popup")) {
+                        return; // Popup already displayed, do nothing
+                    }
+
+                    const overlay = document.createElement("div");
```

Comment:
Isn't it possible to reuse the list of alerts below to add the error about custom JS and perhaps the button also? `alert.appendChild(...)`